src/docker-it/run_docker_sshd.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# requires command line argument with path to rultor root folder
+if ($1); then
+    echo "Required path to rultor root folder"
+    exit 1
+fi
+echo "Path to rultor root folder is $1"
+dockerit=$1/src/docker-it
+container=rultor_sshd
+image="${container}_img"
+echo "Image=$image"
+echo "Container=$container"
+echo "Grant 600 permissions for keys"
+chmod 600 "$dockerit/id_rsa"
+chmod 600 "$dockerit/id_rsa.pub"
+echo "Building docker image $image"
+docker build -t $image "$dockerit"
+echo "Removing docker container $container"
+docker stop "$container"
+docker rm "$container"
+echo "Running docker container $container"
+docker run --name "$container" -d -p 22 "$image"
```

Comment:
@vkuchyn lets not use the name here, this is a little unportable maybe. It does for one not allow running this test twice on the same Docker host, but also there might be other more likely issues from using a fixed name.

Let's just use `cid` files :) Docker even has a nice option for doing that automatically for us: `docker run --cidfile` and then when want to kill the container we can just read the `cid`.

See `_head.sh` and `end` for an existing implementation of this, its a trivial 4 line change, nothing big.